### ISSUE TITLE
BUG: Fourth font not load correctly

### ISSUE THREAD

**Issue by freshchild** [zsviczian/obsidian-excalidraw-plugin#1885]
**Your environment**
```txt
SYSTEM INFO:
	Obsidian version: v1.6.5
	Installer version: v1.5.8
	Operating system: Windows 10 Enterprise N 10.0.22631
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: light
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Excalidraw v2.2.9

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Describe the bug**
When the fourth font is enabled, what is actually displayed seems to be the system default font, not the set fourth font.

**To Reproduce**
1. Create an vault
2. Set the fourth font (Usint <https://github.com/lxgw/LxgwWenKai>)
    ![image](https://github.com/user-attachments/assets/5bc83e02-bb0f-42c2-9366-1501e1f55bdb)
    
    data.json
    
    ```json
    {
        "experimentalEnableFourthFont": true,
        "experimantalFourthFont": "LXGWWenKaiMono-Regular.ttf",
    }
    ```
3. Modify the font family, but the font does not change
    
    ![image](https://github.com/user-attachments/assets/f842cada-c14b-4db0-8515-876af091a3c4)

**Additional context**
For convenience of comparison, the difference between test font (LXGW WenKai Mono) and system default font is attached.

![image](https://github.com/user-attachments/assets/30c3b4c7-cf82-49ae-bbcb-d7c07a4d2275)

**Comment by zsviczian** [comment]
can you reproduce this issue with a font downloaded from google fonts? 
https://fonts.google.com/

I've also experienced that a few font files do not load correctly, I tested your font and I can reproduce the issue... however I don't know what to do about it. Other ttfs load correctly. I am not a font expert and it seems nor is chatGPT. So I recommend looking for a different font. Sorry.

**Comment by freshchild** [comment]
Interesting, google fonts (Noto Sans SC) didn't reproduce, so I checked it further (with [LXGWWenKaiMono-Regular.ttf](https://github.com/lxgw/LxgwWenKai)):

- Set image type as "Native SVG", and got the same wrong output as before
- Check the @font-face of LocalFont, accidentally found the console display `Failed to decode downloaded font: <URL>`
- Check base64, first my tool got the same base64, and then after converting the base64 to hex and saving, I got a file with same SHA512 to the original font file

So it might be an issue of obsidian. But we can "compress" the font size as a temporary solution:

- Changed to a lighter font (e.g. [LXGWWenKaiLite-Regular.ttf](https://github.com/lxgw/LxgwWenKai-Lite/releases)), it loads fine.
- Convert original font to WOFF2 format, it works too.

Or just try to use local fonts like `src: local('LXGW WenKai Mono')`, but I don't know if it's feasible.
